include/sacn/cpp/source.h

```diff
@@ -49,10 +50,12 @@ namespace sacn
 class Source
 {
 public:
+  struct HandleType
```

Comment:
Actually one more thing. I recommend moving this definition out of the Source class so it won't pop up on people's intellisense easily and confuse them. I would put it right above the Source class definition under the sacn namespace and in a detail namespace so it will be out of people's way.

Like so:
```cpp
namespace sacn
{

namespace detail
{
class SourceHandleType {};
};

class Source
{
public:
  using Handle = etcpal::OpaqueId<detail::SourceHandleType, sacn_source_t, SACN_SOURCE_INVALID>;

  // ...
};
```

Reply:
https://github.com/ETCLabs/sACN/pull/45/commits/a3c9e020184c06f758fe78023a716e4ae028437f